packages/react-instantsearch/src/widgets/SortBy/SortBy.js

```diff
@@ -34,12 +34,11 @@ class SortBy extends Component {
 
   render() {
     const {applyTheme, refine, items, selectedIndex} = this.props;
-
     return (
       <Select
-        {...applyTheme('root', 'root')}
+        applyTheme={applyTheme}
```

Comment:
When I first try it, no. Only the SortByLinks. 
Got this : uncaught TypeError: applyTheme is not a function